build.maven/rcptt/pom.xml

```diff
@@ -9,11 +9,9 @@
   <properties>
     <rcptt.version>2.5.1</rcptt.version>
     <designer.os.arch>Windows_x64</designer.os.arch>
-    <deploy.skip>true</deploy.skip>
```

Comment:
I'm not sure if it's the best if we make this rcptt test project available over the importer. Because this is not really a designer project...

Reply:
Hm... this is anyway triggered from another build (or maven lifecycle) where no deployment is executed... So a useless change?

Reply:
- yes rcptt is in own jenkins build
- and deploy.skip has no affect. because this property was not used nowhere. the real name would be maven.deploy.skip.

Reply:
and yes - rcptt should not be deployed to anywhere.